docs/services/slam/cartographer/_index.md

```diff
@@ -193,14 +130,19 @@ Note that [Data Capture](/services/data/capture/) continuously monitors and capt
 
    Check the **LOGS** tab of your machine in the Viam app to make sure your RPlidar has connected and no errors are being raised.
 
+3. (Optional) Configure cartographer to use cloudslam:
+
+    On the `SLAM` service configuration pane, switch the configuration to the Advanced view and set the `use_cloud_slam` field to **true**. This setting disables local mapping to limit cpu usage in favor of using cloudslam.
+
+    In addition, your configured LiDAR camera and movement sensor must have data capture enabled. See the [cloudslam](../cloudslam) documentation for more information on how to configure the feature on your machine and how to use cloudslam.
+
 {{% /tab %}}
 {{% tab name="JSON Example" %}}
 
 This example JSON configuration:
 
 - adds the `viam:rplidar` and the `viam:cartographer` modules
-- configures the `viam:slam:cartographer` service and the [data management service](/services/data/)
```

Comment:
so what now happens with the data service?

Reply:
the data service is only required when using cloudslam, so I wanted to move any references to data into that doc. Cartographer running locally has no dependencies on data capture

Reply:
ok thx